ironfish/src/wallet/wallet.test.ts

```diff
@@ -933,6 +933,109 @@ describe('Accounts', () => {
     })
   })
 
+  describe('burn', () => {
+    it('returns a transaction with matching burn descriptions', async () => {
+      const { node } = await nodeTest.createSetup()
+      const account = await useAccountFixture(node.wallet)
+
+      const mined = await useMinerBlockFixture(node.chain, 2, account)
+      await expect(node.chain).toAddBlock(mined)
+      await node.wallet.updateHead()
+
+      const asset = new Asset(account.spendingKey, 'mint-asset', 'metadata')
+      const mintValue = BigInt(10)
+      const mintBlock = await useBlockFixture(node.chain, async () => {
```

Comment:
you could use my new useMintBlockFixture from my other open PR https://github.com/iron-fish/ironfish/pull/2810

Reply:
Updated